client/src/consts/theme.ts

```diff
@@ -0,0 +1,13 @@
+import {css} from 'styled-components/native';
```

Comment:
We could use https://www.styled-components.com/docs/advanced#theming but do we really need it?